oonib/testhelpers/http_helpers.py

```diff
@@ -168,6 +172,14 @@ def all(self, length, keyword):
             length = 100000
         self.write(self.genRandomPage(length, keyword))
 
+class TCPConnectProtocol(Protocol):
+    def connectionMade(self):
+        self.transport.loseConnection()
+
+class TCPConnectFactory(Factory):
+    def buildProtocol(self, addr):
+        return TCPConnectProtocol()
+
```

Comment:
Yeah you are right this is a leftover from the cherry-ditching of the web_connectivity test helper.